addons/folo/jaxrs/src/main/java/org/commonjava/indy/folo/bind/jaxrs/FoloAdminResource.java

```diff
@@ -345,4 +353,54 @@ public Response importReport( final @Context UriInfo uriInfo, final @Context Htt
         return types;
     }
 
+    @ApiOperation( "Batch delete files uploaded through FOLO trackingID under the given storeKey." )
+    @ApiResponse( code=200, message = "Batch delete operation finished." )
+    @ApiImplicitParam( name = "body", paramType = "body",
+                    value = "JSON object, specifying trackingID and storeKey, with other configuration options",
+                    required = true, dataType = "org.commonjava.indy.model.core.DemoteRequest" )
+    @Path( "/batch/demote" )
+    @POST
+    public Response doDelete( @Context final UriInfo uriInfo, final DemoteRequest request )
+    {
+
+        String trackingID = request.getTrackingID();
+
+        if ( trackingID == null || request.getStoreKey() == null )
```

Comment:
I need to delete some test repos and they are not in folo (some are, most are not because failures). In this case, the only input is storeKey. anyway, that is something different. I will submit another pr.